src/logo-blau.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
each logo has its own file with a default export (this is required for lazy imports)